tools/browse_dataset.py

```diff
@@ -0,0 +1,51 @@
+import argparse
+import tempfile
+
+import mmcv
+from mmcv import Config
+from tqdm import tqdm
+
+from mmdet.datasets.builder import build_dataset
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description='Train a detector')
+    parser.add_argument('config', help='train config file path')
+    parser.add_argument('--skip-type', nargs='+', default=['DefaultFormatBundle'])
+    args = parser.parse_args()
+    return args
+
+
+def clear_config(config_path, skip_type):
+    c = open(config_path).read()
+    c = c.replace('to_rgb=True', 'to_rgb=False') \
+        .replace('mean=[123.675, 116.28, 103.53], std=[58.395, 57.12, 57.375]', 'mean=[0] * 3, std=[1] * 3') \
+        .replace("dict(type='Normalize', **img_norm_cfg),",
+                 """dict(type='Normalize', **img_norm_cfg),
+                    dict(type='ToUint8'),""")
+    for s in skip_type:
+        c = c.replace(s, 'Pass')
+    print(c)
+    ft = tempfile.NamedTemporaryFile('w+t', suffix='.py', delete=False)
+    print(ft.name)
+    ft.write(c)
+    ft.close()
```

Comment:
The logic and code of this function seem not so good. 
Renamed this func as `retrieve_data_cfg`.
The logic can be  as follow:
1. Read config by `cfg = Config.fromfile(args.cfg)`
2. Retrieve cfg of `train.data.train` as `train_data_cfg`, which contains `pipeline` field.
3. Delete unused pipeline like `Normalize, DefaultFormatBundle, Collect`.